### ISSUE TITLE
tests fail 

### ISSUE THREAD

**Issue by bryanhuntesl** [bitwalker/swarm#74]
I perform checkout of master and run the tests, the following failures are generated:

```
14:11:12.342 [info]  Protocol 'inet_tcp': register/listen error: econnrefused

** (EXIT from #PID<0.73.0>) :not_alive

14:11:12.361 [error] Task #PID<0.150.0> started from #PID<0.73.0> terminating
** (stop) :not_alive
    (stdlib) slave.erl:198: :slave.start/5
    (swarm) test/support/cluster.ex:21: Swarm.Cluster.spawn_node/1
    (elixir) lib/task/supervised.ex:88: Task.Supervised.do_apply/2
    (elixir) lib/task/supervised.ex:38: Task.Supervised.reply/5
    (stdlib) proc_lib.erl:247: :proc_lib.init_p_do_apply/3
Function: #Function<8.128785774/0 in Swarm.Cluster.spawn/1>
    Args: []

14:11:12.362 [error] Task #PID<0.151.0> started from #PID<0.73.0> terminating
** (stop) :not_alive
    (stdlib) slave.erl:198: :slave.start/5
    (swarm) test/support/cluster.ex:21: Swarm.Cluster.spawn_node/1
    (elixir) lib/task/supervised.ex:88: Task.Supervised.do_apply/2
    (elixir) lib/task/supervised.ex:38: Task.Supervised.reply/5
    (stdlib) proc_lib.erl:247: :proc_lib.init_p_do_apply/3
Function: #Function<8.128785774/0 in Swarm.Cluster.spawn/1>
    Args: []
```

As with libcluster, is there some setup I need to perform prior to running the tests?

**Comment by mbaeuerle** [comment]
I don't get the same errors but when trying to run `mix test` most of the tests fail with `{:error, :no_node_available}`. A test setup guide would definitely help.

**Comment by bitwalker** [comment]
Addressed in #94
